assets/csi_cloud_config.ini

```diff
@@ -6,4 +6,5 @@ cluster-id = "${CLUSTER_ID}"
 [VirtualCenter "${VCENTER}"]
 insecure-flag = "true"
 datacenters = "${DATACENTERS}"
+migration-datastore-url = "${MIGRATION_DATASTORE_URL}"
```

Comment:
Where will this datastore will be used?

Reply:
This datastore will be used when no datastore is set in intree SC and rather than current behaviour of randomly picking a datastore from all available datastores, the datastore specified here will be used.